### ISSUE TITLE
Drop sub menus can't be reached via keyboard navigation

### ISSUE THREAD

**Issue by JammingBen** [opencloud-eu/web#2097]
### Initial Checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Bug Description

Drop sub menus (e.g. "Open with") can't be reached via keyboard navigation.

### Reproduction Steps

- create a folder
- right click the folder
- try opening the "Open with" sub menu via keyboard navigation

### Expected Outcome

The sub menu can be opened via keyboard navigation. Ideally, the drop menu can be navigated via arrow keys.

### Actual Outcome

There is no way to reach the sub drop menu via keyboard navigation.
